src/action.ts

```diff
@@ -64,6 +65,34 @@ export const runAction = async (space): Promise<void> => {
     count++;
   }
 
+  if (WITH_DELIVERY_TOKEN) {
+    Logger.log("Creating new CDA token for ephemeral environment");
+    var branchName = branchNames.headRef
+
+    space.createApiKey({
```

Comment:
what if the token already exists from a previous run?

example:

1. open pull request => 1st GHA run
2. push a commit to address a comment => 2nd GHA run

Reply:
excellent question. 
It would fail the second time with warning message. I believe it would still work, but I agree it is NOT pretty 👍 
This forced me to review again logic of this workflow and re-do token creation a bit differently.